components/authz-service/server/v2/system.go

```diff
@@ -48,6 +48,12 @@ func SystemPolicies() []*storage.Policy {
 				Resources: []string{"system:service:version"},
 				Projects:  []string{constants.AllProjectsID},
 			},
+			{
+				Effect:    storage.Allow,
+				Actions:   []string{"system:telemetryConfig:get"},
```

Comment:
everyone should have this access, and we were inadvertently denying it to all users, and providing it to certain roles by accident due to the `system:*` namespace.